docs/messageData.md

```diff
@@ -2920,8 +2947,107 @@ The two organizers meet in person to exchange the federation_details message (th
 
 They also decide on a time slot to perform their next roll-calls---these have to happen simultaneously.
 
+## Requesting a challenge (federation#challenge_request)
+The organizer of one of the two LAOs will request a challenge from his server.
+This challenge will be then used by the other server to authenticate itself.
+
+<details>
+<summary>
+💡 See an example
+</summary>
+
+```json5
+// ../protocol/examples/messageData/federation_challenge_request/federation_challenge_request.json
+{
+  "object": "federation",
+  "action": "challenge_request",
+  "timestamp": 1712854874
+}
+
+```
+</details>
+
+```json5
+// ../protocol/query/method/message/data/dataFederationChallengeRequest.json
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://raw.githubusercontent.com/dedis/popstellar/master/protocol/query/method/message/data/dataFederationChallengeRequest.json",
+  "description": "Sent by an organizer client to its server, to retrieve a challenge object",
+  "type": "object",
+  "properties": {
+    "object": {
+      "const": "federation"
+    },
+    "action": {
+      "const": "challenge_request"
+    },
+    "timestamp": {
+      "type": "integer",
+      "description": "[Timestamp] of the request",
+      "minimum": 0
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "timestamp"
```

Comment:
I believe the `object` and `action` field are missing there.

Reply:
These fields are missing in the file ../protocol/query/method/message/data/dataFederationChallengeRequest.json